scrapy/extensions/memusage.py

```diff
@@ -92,6 +92,8 @@ def _check_limit(self):
                 self.crawler.engine.close_spider(self.crawler.engine.spider, 'memusage_exceeded')
             else:
                 self.crawler.stop()
+        else:
+            logger.info("Current memory usage is %(virtualsize)dM", {'virtualsize': self.get_virtual_size()})
```

Comment:
Rather than calling `self.get_virtual_size()` twice, why not declare a variable with its value at the beginning of the method, and use that value here and on the `if` statement?

Also, you are logging bytes as megabytes. You need to do `size / 1024 / 1024` as done above with the `mem` variable, but for the outcome of `self.get_virtual_size()`.

Also, for precision, maybe we should change the existing logging messages and your own message to use `MiB` instead of `M` as unit for [mebibytes](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units).

Reply:
> If so, this would just require the label be changed in the logging.

Yes, my point is that we are logging the value as e.g. 100M, but `M` is not a valid unit, it should be MB or, better yet, MiB.

But it is an issue from the original code, so I am OK with not addressing it here.

Reply:
Makes sense to me. Thank you for the clarification